Github/service.js

```diff
@@ -167,3 +168,82 @@ module.exports.getAllCommitsTogether = branches => {
       });
   });
 };
+
+const getIssues = repo => {
+  return new Promise((resolve, reject) => {
+    request.get(
+      {
+        url: `https://api.github.com/repos/Seneca-CDOT/${repo}/issues?labels=help+wanted&access_token=${key}`,
+        headers: { 'User-Agent': 'request' },
```

Comment:
Seems like `request` package supports `qs` parameter with queryistring values (see https://www.npmjs.com/package/request#requestoptions-callback)

Instead of constructing the url links ourselves as `?paramOne=valueOne&paramTwo=valueTwo...` we could pass it with `request.get` as `qs : {'paramOne': 'valueOne' ...}`